### ISSUE TITLE
Initial Cleanup Items

### ISSUE THREAD

**Issue by jhiemstrawisc** [osg-htc/osdf-website#4]
# Screenshot 1
<img width="640" alt="osdf-website-pic-1" src="https://github.com/user-attachments/assets/fd0f587b-8f25-471e-9b19-948978e5a943">\
- The left card in this screeshot mentions CVMFS -- I doubt we want to mention that.
- I think the right card should say something like "The [Pelican client](link to wherever we want to point for docs, probably https://docs.pelicanplatform.org/getting-data-with-pelican/client) is available for every operating system"

# Screenshot 2
<img width="640" alt="osdf-website-pic-2" src="https://github.com/user-attachments/assets/2524c866-d961-4372-98c2-9e5693bed5c3">\
- Replace all mentions of `StashCP` with Pelican. Most of the actual "StashCP does XYZ" stuff is still true, it's just called something different now
- The intro that says "StashCP is the client tool to download or upload files on the OSDF". I wonder how we can rephrase this so that it still says Pelican is the client tool to do stuff both with the OSDF _and_ with other data federations? Maybe we're find only mentioning OSDF here, since this is the OSDF website...
- Update download link for ~~stashcp~~pelican to point at the appropriate spot. I don't know much about what you're cooking up with a download server, but maybe that's the way to go?
- Update the "View the Source" link to point to https://github.com/pelicanplatform/pelican
- Replace the terminal image with the command `pelican object get pelican://osg-htc.org/ospool/uc-shared/public/OSG-Staff/validation/test.txt .

# Screenshot 3
![Screenshot 2024-11-26 at 10 56 29](https://github.com/user-attachments/assets/ed1cb63f-c92d-4678-9468-98d27fddf889)
- The OAC number here is for PATh, and I think we also want to include the Pelican grant number. A quick chat with BrianB, and I think we want this to be:
```
This project is supported by National Science Foundation under Grant [OAC-2331480](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2331480) and Cooperative Agreement [OAC-2030508](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2030508). Any opinions, findings, conclusions or recommendations expressed in this material are those of the authors and do not necessarily reflect the views of the National Science Foundation.
```
I'm told the language difference between "Grant" (what Pelican is) and Cooperative Agreement (what PATh is) is important here.

# Screenshot 4
![Screenshot 2024-11-26 at 11 03 22](https://github.com/user-attachments/assets/e0601803-cb0b-4ece-83b2-993bc3157b42)
- The ToC links in the top right seem okay except for the Docs index. I think this should point to https://docs.pelicanplatform.org

**Comment by djw8605** [comment]
> The ToC links in the top right seem okay except for the Docs index. I think this should point to https://docs.pelicanplatform.org/

Is this an OSDF website, or pelican?  The rest of the webpage is very much just about the osg-htc.org federation (for example, on the globe)

**Comment by CannonLock** [comment]
@djw8605 Good point, I will adjust the acknowledgement to include both.

**Comment by jhiemstrawisc** [comment]
> > The ToC links in the top right seem okay except for the Docs index. I think this should point to https://docs.pelicanplatform.org/
> 
> Is this an OSDF website, or pelican? The rest of the webpage is very much just about the osg-htc.org federation (for example, on the globe)

@djw8605 fair enough, but my main concern is that a lot of the `osdf.osg-htc.org` page talks about osdf clients, while the `https://osg-htc.org/docs/data/stashcache/overview/` page linked to by Docs is specific to setting up new services (and has no client documentation). If we want to keep linking there, then maybe that page should have a link to client usage docs? We should probably also remove "stashcache" from the path as well, but I don't know what that breaks for other people who have the link embedded...
